packages/next/telemetry/events/swc-plugins.ts

```diff
@@ -1,4 +1,6 @@
+import { existsSync } from 'fs'
```

Comment:
Should we use our file-exists util instead? https://github.com/vercel/next.js/blob/canary/packages/next/lib/file-exists.ts

Reply:
Yes, looks like my oversight. Thanks for pointing out.